src/templater.rb

```diff
@@ -42,6 +42,17 @@ class Templater
     # `metal configure` commands.
     EOF
 
+    MANAGED_START = 'METALWARE_START'
+    MANAGED_END = 'METALWARE_END'
+    MANAGED_START_COMMENT = "# #{MANAGED_START}"
+    MANAGED_END_COMMENT = "# #{MANAGED_END}"
```

Comment:
Could we make these comments stand out a bit more? If the managed section is long, then it could be easily missed when scrolling through.
```
Currently:
# METALWARE_START

Maybe:
######### METALWARE_START #########
```